Jenkinsfile

```diff
@@ -18,60 +18,66 @@ node('linux && docker') {
         sh 'npm run build'
       }
 
-      stage('Install Chrome') {
-        sh "wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -"
-        sh "echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | tee /etc/apt/sources.list.d/google-chrome.list"
-        sh "apt-get update"
-        sh "apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 libxtst6 --no-install-recommends"
-        sh "google-chrome --version"
-      }
-
       stage('Linting') {
         sh 'npm run lint:check'
       }
 
+      stage('Generate Docs') {
+        sh 'npm run doc:generate'
+      }
+
       stage('Unit Test') {
         sh 'npm test'
         junit 'packages/*/reports/*.xml'
       }
```

Comment:
Perhaps we could also run unit tests under the same conditions, then.

Reply:
We could if we want to speed up the build even more, hmm. Since UTs have not been flaky, I kept them to have some tests running prior to deploying.

I think I will merge the PR as is for now, and see how things go. If we want to speed up the build even more, we can revisit,